partner_credit_limit_history/wizard/partner_credit_limit_wizard.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PartnerCreditLimitWizard(models.TransientModel):
+    _name = "partner.credit.limit.wizard"
+    _description = "Partner Credit History"
+
+    partner_id = fields.Many2one(
+        comodel_name="res.partner", string="Partner", required=True
+    )
+    amount = fields.Float(string="New Amount", required=True)
+    note = fields.Char(string="Reason", required=True)
+
+    def action_confirm(self):
+        """We will need to apply sudo because regular user does not have
```

Comment:
This should be an inline comment, not as docstring, as this is not talking about the goal of the method, but the specific `sudo` operation. Not blocking this time anyways.